### PR TITLE
Feature/question pagination

### LINKED ISSUE #8: 007-feature-pagination-questions
Activates pagination in admin-questions+

### PR DESCRIPTION
**Pull request by ApostolaT**
Add pagination for the questions on admin-question page

### CODE REVIEW

File: src/Services/QuestionService.php [hunk 1]
```diff
@@ -4,17 +4,35 @@
...
+    private const RESULTS_PER_PAGE = 10;
+
     private $repositoryManager;
 
     public function setRepositoryManager(RepositoryManagerInterface $repositoryManager)
     {
         $this->repositoryManager = $repositoryManager;
     }
+    /**
+     * Counts how many user entities the Repository has.
```
Comment (ApostolaT) [inline src/Services/QuestionService.php]: I changed this and it will be present in the next commit

File: src/Controllers/ResultController.php [hunk 2]
```diff
@@ -18,6 +18,7 @@ class ResultController extends AbstractController
     /**
      * @var AbstractService
      */
+    //TODO change to private
```
Comment (ApostolaT) [inline src/Controllers/ResultController.php:21]: This is for the future refactoring feature

File: src/Controllers/QuestionController.php [hunk 3]
```diff
@@ -77,7 +90,7 @@ public function deleteQuestion(RequestInterface $request)
             if ($this->questionService->delete($request)) {
                 $response = new Response(Stream::createFromString(' '), []);
                 $response = $response->withStatus(301);
-                $response = $response->withHeader('Location', 'http://local.quiz.com/question/1');
+                $response = $response->withHeader('Location', 'http://local.quiz.com/question');
 
                 return $response;
```
Comment (ApostolaT) [inline src/Controllers/QuestionController.php:95]: i will use the getRedirectPage method from the abstract controller, but now it's not the goal of the current feature

Comment (victortodoran) [reply src/Controllers/QuestionController.php:95]: ok

File: src/Services/QuestionService.php (hunk 1 above)
Comment (victortodoran) [reply src/Services/QuestionService.php]: return type

File: src/Services/QuestionService.php [hunk 4]
```diff
@@ -23,15 +41,20 @@ public function selectAll()
         return $repository->findBy([], [], 0, 0);
     }
 
-    public function getAll(RequestInterface $request)
+    /**
+     * Gets all the question from the specified page
+     * @param int $page
+     * @return |null
+     */
```
Comment (victortodoran) [inline src/Services/QuestionService.php:48]: return type

**Pull request merged**